Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -1970,6 +1970,45 @@ script:
     - contextPath: ServiceNow.Ticket.Note.Type
       description: Ticket note type
       type: string
+  - arguments:
+    - default: false
+      description: Ticket System ID
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    - default: false
+      description: 'Tag system ID. Can be retrieved using: "!servicenow-query-table
```

Comment:
The table is `label_entry`

Reply:
And we also need the sys_id field naturally

Reply:
The tags are in the `label` table, but the tags that were added to incidents are in the `label_entry`. editting the descriptions accordingly.